tests/test_gtfs_objects.py

```diff
@@ -0,0 +1,251 @@
+    assert len(routes.fare_attrs_df) == 9
+    match_fare_rule_attr_df = routes.fare_rules_df[routes.fare_rules_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD].isin(routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])]
+    assert len(match_fare_rule_attr_df) == len(routes.fare_rules_df)
+    #pd.testing.assert_frame_equal(routes.fare_rules_df[fare_attrs_df_dtype.keys()].drop_duplicates().set_index(Route.FARE_ATTR_COLUMN_FARE_PERIOD),
+    #                              routes.fare_attrs_df[routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD].isin(match_fare_rule_attr_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])].set_index(Route.FARE_ATTR_COLUMN_FARE_PERIOD),
+    #                              check_dtype=False, check_index_type=False)
+
+    #routes.fare_ids_df
+    assert not routes.fare_ids_df.empty
+    pd.testing.assert_frame_equal(routes.fare_rules_df[[Route.FARE_RULES_COLUMN_FARE_ID, Route.FARE_RULES_COLUMN_FARE_ID_NUM]].drop_duplicates().set_index(Route.FARE_RULES_COLUMN_FARE_ID_NUM),
+                                  routes.fare_ids_df.set_index(Route.FARE_RULES_COLUMN_FARE_ID_NUM))
+
+    #routes.fare_by_class
+    assert routes.fare_by_class
+
+    #routes.fare_transfer_rules_df
+    assert not routes.fare_transfer_rules_df.empty
+    assert len(routes.fare_transfer_rules_df[routes.fare_transfer_rules_df[Route.FARE_TRANSFER_RULES_COLUMN_FROM_FARE_PERIOD]
+        .isin(routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])]) == len(routes.fare_transfer_rules_df)
+    assert len(
+        routes.fare_transfer_rules_df[routes.fare_transfer_rules_df[Route.FARE_TRANSFER_RULES_COLUMN_TO_FARE_PERIOD]
+        .isin(routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])]) == len(routes.fare_transfer_rules_df)
+
+
+@pytest.mark.skip(reason='Need to upgrade to Partridge')
+def test_transfers_load(network_dir, gtfs_feed):
+    transfers = Transfer(network_dir, os.path.join(HOME_DIR, 'test_gtfs_objects'),
+                         gtfs_feed)
+
+    assert not transfers.transfers_df.empty
+    transfers_df_dtypes = {
+        'min_transfer_time': np.float64,
+        'from_stop_id': object,
+        'to_stop_id': object,
+        'transfer_type': np.float64,
+        'stop2stop': bool,
+        'dist': np.float64,
+        'from_route_id': object,
+        'to_route_id': object,
+        'elevation_gain': np.int64,
+        'schedule_precedence': object,
+        'transfer_penalty': np.float64,
+        'min_transfer_time_min': np.float64,
+        'time_min': np.float64,
+        'time': np.dtype('timedelta64[ns]'),
+    }
+    assert (set(transfers_df_dtypes.keys()).issubset(transfers.transfers_df))
+    assert transfers.transfers_df.dtypes.to_dict() == transfers_df_dtypes
+    assert len(transfers.transfers_df) == 6
+
+    assert len(transfers.transfers_df[transfers.transfers_df['min_transfer_time'] > 0]) == 1
+
+@pytest.mark.skip(reason='Need to upgrade to Partridge')
+def test_trips_load(network_dir, gtfs_feed):
```

Comment:
add description

Reply:
Cleaned up test_gtfs_objects to be compatible with Partridge. Added docstrings.